tracer/src/Datadog.Trace/Telemetry/TelemetryProductType.cs

```diff
@@ -0,0 +1,15 @@
+﻿// <copyright file="TelemetryProductType.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Telemetry;
+
+internal enum TelemetryProductType
+{
+    AppSec,
+    Profiler,
+    DynamicInstrumentation,
```

Comment:
Question for telemetry team rather than you, but what qualifies as a product? 
I'd say we miss `Tracer` and `CIVisibility` here (even though maybe we don't send telemetry for the first one, and we already have the info for the other one).
What about smaller products like DBM,DSM?

Reply:
Dunno 🤷  😄  These are the only allowed values